modules/deleting-a-namespace-using-the-web-console.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_install/uninstalling-container-native-virtualization.adoc
+
+[id="deleting-a-namespace-using-the-web-console_{context}"]
+= Deleting a namespace using the web console
+
+.Procedure
+
+. Navigate to *Administration* -> *Namespaces*.
+
+. Locate the namespace that you want to delete in the list of namespaces.
+
+. On the far right side of the namespace listing, select *Delete Namespace* from the
+Options menu {kebab}. If you do not have permissions to delete the namespace, the *Delete Namespace*
+option is grayed out and the option is not clickable.
+
+. When the *Delete Namespace* pane opens, enter the name of the namespace that
+you want to delete in the field.
+
+. Click *Delete*.
```

Comment:
I'm guessing this removes all resources associated with the namespace. Might also be worthwhile including in an intro para.

Reply:
I'm checking with Irina and will add this info when I hear back :)

Reply:
After discussing with @alexxa I am going to merge this as-is.